config-vm.md

```diff
@@ -0,0 +1,37 @@
+# Virtual-machine-specific Configuration
+
+Virtual-machine-based runtimes require additional configuration to that specified in the [default spec configuration](config.md).
+
+This optional configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) host path to file that represents the root filesystem for the virtual machine.
```

Comment:
@jodh-intel I am not quite sure here. This isn't exactly the right layer to be relying on autodetect. That seems like something that would belong at a higher-level (ie whatever creates this file). Containers are mostly different in that the rootfs is either pre-expanded or a few host mounts away.

With an image file, its quite different. Does this image file get mounted? How are the parameters for that mount resolved? Is `kernel.path` relative to the image root or to the host root? What happens to the regular container `rootfs`? Does it get replaced by this image?